server/jetstream_cluster.go

```diff
@@ -153,6 +153,24 @@ const (
 	defaultMetaFSBlkSize = 1024 * 1024
 )
 
+// Returns information useful in mixed mode.
+func (s *Server) trackedJetStreamServers() (js, total int) {
+	s.mu.Lock()
+	defer s.mu.Unlock()
+	if !s.running || !s.eventsEnabled() {
+		return -1, -1
+	}
+	s.nodeToInfo.Range(func(k, v interface{}) bool {
+		si := v.(nodeInfo)
+		if si.js {
+			js++
+		}
+		total++
+		return true
+	})
+	return js, total
```

Comment:
Does this contain nodes we have seen only?
meaning total may not be the value started with as some servers have not been observed?

Reply:
Correct, only nodes we have seen and update for are counted here so we can not go down based on split brain scenario on cold boot.